src/LykkePublicAPI/Controllers/MarketController.cs

```diff
@@ -41,6 +41,44 @@ public class MarketController : Controller
         /// <returns></returns>
         [HttpGet]
         public async Task<IEnumerable<ApiMarketData>> Get()
+        {
+            return await _cache.TryGetFromCacheAsync(
+                "Market:Overall", 
+                async () => await GetMarketInternal(),
+                absoluteExpiration: TimeSpan.FromMinutes(30));
+        }
+
+        /// <summary>
+        /// Get trade volume for asset pair
+        /// </summary>
+        [HttpGet("{assetPair}")]
+        public async Task<ApiMarketData> Get(string assetPair)
+        {
+            return await _cache.TryGetFromCacheAsync(
+                $"Market:AssetPairs:{assetPair}", 
+                async () => await GetMarketInternal(assetPair), 
+                absoluteExpiration: TimeSpan.FromMinutes(30));
```

Comment:
TimeSpan.FromMinutes(30) let's move this in config?

Reply:
This is excess, since this is consequence of the data calculation algorith, which updates data once per hour